src/main/java/ch/wisv/areafiftylan/dto/EventDTO.java

```diff
@@ -0,0 +1,28 @@
+package ch.wisv.areafiftylan.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Created by Sille Kamoen on 27-3-16.
+ */
+public class EventDTO {
+
+    @Getter
+    @Setter
+    @NotEmpty
+    String name = "";
+
```

Comment:
What about some more fields? Such as a location of an image for on the site? Maybe a description, sponsor, prices? We could even think of some more.